InfiniteLoopers-App/src/main/java/com/isa/menu/Menu.java

```diff
@@ -0,0 +1,78 @@
+package com.isa.menu;
+
+import java.util.List;
+import java.util.Scanner;
+import static com.isa.menu.OptionService.backToMainMenu;
+
+
+public class Menu {
+
+    private Menu() {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+    }
+        public static void menuInvoke() {
+            Scanner scanner = new Scanner(System.in);
+            List<Option> optionList = OptionService.getOptionList();
+
+
+            System.out.println("**************************************************");
+            System.out.println("                Dziennik XXXXXXXX                 ");
+            System.out.println("**************************************************");
+            System.out.println("\n");
+
+            for (Option x : optionList) {
```

Comment:
raczej nie chcemy nazywać zmiennych `x` :)

Reply:
Mam jeszcze czasem problem z nazewnictwem. Jesli zmienna jest tylko w metodzie to nie moge tak robic?

Reply:
mimo wszystko lepiej to nazwać po prostu `option`